frontend/src/routes.tsx

```diff
@@ -2,15 +2,19 @@ import React from "react";
 import { Route } from "react-router-dom";
 
 import Login from "containers/LoginView";
-import Posts from "containers/PostsView";
+import PostsList from "containers/PostsList";
 import Signup from "containers/SignupView";
+import PostDetail from "containers/PostDetail";
+import PostCreationView from "containers/PostCreationView";
 
 const BaseRouter = () => (
-  <div>
-    <Route exact path="/" component={Posts} />{" "}
-    <Route exact path="/login" component={Login} />{" "}
-    <Route exact path="/signup" component={Signup} />{" "}
-  </div>
+    <div>
+        <Route exact path="/" component={PostsList}/>{" "}
+        <Route exact path="/posts/new/" component={PostCreationView}/>{" "}
```

Comment:
is this {" "} added automatically by prettier or smth?

Reply:
To be honest, I'm not sure what this is.